packages/compass-schema/src/components/field/field.jsx

```diff
@@ -181,13 +173,18 @@ class Field extends Component {
       <div className={cls}>
         <div className="row">
           <div className="col-sm-4">
-            <div className="schema-field-name" onClick={this.titleClicked.bind(this)}>
+            {/* eslint-disable jsx-a11y/click-events-have-key-events */}
+            {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
```

Comment:
There are a few other occurrences where we disable these eslint rules (we should fix them at some point). I didn't want to make those changes here as they would involve more changes to the components. I'm thinking it would be too much for one pr.

Reply:
I am really afraid that if we are not addressing these right away and just plan to do this at some point, we will not get to fixing these ever, it's low effort to update and I think can be done as a follow up in the same ticket scope. This doesn't only apply to a11y rules, but all the rules that eslint might highlight during these migrations. If you even updated some outdated usage of react lifecycles, why are we specifically excluding a11y fixes from the project?

Reply:
Opened a separate ticket so we can be more explicit about the change and include it in the changelog. Put it in progress. [COMPASS-5758](https://jira.mongodb.org/browse/COMPASS-5758)